featurex/transformers.py

```diff
@@ -54,6 +64,42 @@ def transform(self, stims, *args, **kwargs):
         return self._transform(self._validate(stims), *args, **kwargs)
 
 
+def get_converter(in_type, out_type):
```

Comment:
Did you try calling `get_transformer` with `base=Converter`? I haven't tested it, but in principle, it should work the same way.